.npmignore

```diff
@@ -14,4 +14,5 @@ node_modules
 /packages/**/lib/*
 /.nyc_output
 package-lock.json
-.git
\ No newline at end of file
+.git
+!dist/*
```

Comment:
I'm not sure why we need to include this explicitly.
Based on [Documentation of npm-js](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package), dist directory is not the directory that is excluded by default.
Could you check that it worked?

Reply:
When i checked original '.npmignore' file, after install caver-js from npm, i checked dist folder is not included.

So i searched how to include this, and people suggested this solution or describe 'file' section in package.json to include.
I checked dist folder is included with download from npm which is published with this configuration.

Reply:
got it. I'm not sure but it seems that the npm publish also checks .gitignore.
Anyway, please go ahead. :)